assessment.md

```diff
@@ -27,23 +27,23 @@ when creating python programs.
     and development tools
   * G = Additional development tools configured, in addition to minimum required
     dependencies and tools
-  * E = Additional development tools configured and able to provide assistance
-    and troubleshooting to other students regarding their own development
+  * E = Additional development tools configured and ability to provide assistance
+    and troubleshooting to other individuals regarding their own development
     environment
```

Comment:
I would say either:
...additional tools configured and is able to...
or
...additional tools configured and has the ability to...

Reply:
We should remove "being able to" since this assessment is about what has been done rather than what can be done. We should not be based on being able to help others as this would be difficult to prove and unnecessary.